db_migrations.go

```diff
@@ -1993,4 +1993,42 @@ var migrations = []any{
 	newSqlMigration(`
         DROP TABLE network_client_connection_error
     `),
+
+	newSqlMigration(`
+		CREATE TABLE network_user_auth_sso
+		(
+			user_id uuid NOT NULL,
+			auth_type varchar(32) NOT NULL,
+			auth_jwt text NOT NULL,
+			user_auth varchar(256) NOT NULL,
+			create_time timestamp NOT NULL DEFAULT now(),
+
+			PRIMARY KEY (user_id, auth_type)
+		)
+	`),
+	newSqlMigration(`
+		CREATE TABLE network_user_auth_password
+		(
+			user_id uuid NOT NULL,
+			user_auth varchar(256) NOT NULL,
+			auth_type varchar(32) NOT NULL,
+			password_hash bytea NULL,
+            password_salt bytea NULL,
+            verified bool NOT NULL DEFAULT false,
+			create_time timestamp NOT NULL DEFAULT now(),
+
+			PRIMARY KEY (user_id, auth_type)
+		)
+	`),
+	newSqlMigration(`
+		CREATE TABLE network_user_auth_wallet
+		(
+			user_id uuid NOT NULL,
+			wallet_address text NOT NULL,
+			blockchain varchar(32) NOT NULL,
+			create_time timestamp NOT NULL DEFAULT now(),
+
+			PRIMARY KEY (user_id)
+		)
+	`),
```

Comment:
Is the plan to drop `network_user` columns after migration? auth_type, user_auth, password_hash, password_salt, auth_jwt, validated?

Reply:
@xcolwell yup, after migrations, will drop columns from `network_user`